promql/parser/parse_test.go

```diff
@@ -4823,11 +4823,6 @@ var testExpr = []struct {
 			PosRange: posrange.PositionRange{Start: 0, End: 20},
 		},
 	},
-	{
```

Comment:
I would leave this, it's a regression test

Reply:
Done, I thought it had some issue, but it turned out that parser.NewParser had a bug that made the tests interfere with each other.